x-pack/legacy/plugins/lens/server/usage/task.ts

```diff
@@ -71,14 +77,16 @@ function scheduleTasks(server: Server) {
     // function block.
     (async () => {
       try {
-        await taskManager!.schedule({
-          id: TASK_ID,
-          taskType: TELEMETRY_TASK_TYPE,
-          state: { byDate: {}, suggestionsByDate: {}, saved: {}, runs: 0 },
-          params: {},
-        });
+        if (taskManager) {
```

Comment:
I'd lift this up to line 67 or thereabouts, so we don't even register an afterPluginsInit if there is no task manager.